v0_5/src/reset_checkpoints.rs

```diff
@@ -212,7 +212,7 @@ pub trait ResetCheckpointsModule {
         let total_active = self.fund_view_module().get_user_stake_of_type(USER_STAKE_TOTALS_ID, FundType::Active);
         let total_unstaked = self.fund_view_module().get_user_stake_of_type(USER_STAKE_TOTALS_ID, FundType::UnStaked);
 
-        let max_available = &total_active + &total_waiting;
+        let max_available = &(&total_active + &total_waiting) + &total_unstaked;
```

Comment:
why not simply max_available = &total_active + &total_waiting + &total_unstaked ?

Reply:
&BigUint + &BigUint = BigUint (no ref), and we don't have BigUint + &BigUint implemented yet.
I can add them, especially since I have a nice optimization in mind in this case, it wasn't a priority yet.
Note that rust BigUint also doesn't have this implemented.